src/engine/channels/enginedeck.cpp

```diff
@@ -360,10 +359,7 @@ void EngineDeck::slotPassthroughChangeRequest(double v) {
 // static
 QString EngineDeck::getGroupForStem(const QString& deckGroup, int stemIdx) {
     DEBUG_ASSERT(deckGroup.endsWith("]") && stemIdx < 4);
-    QString stemGroup =
-            QStringView(deckGroup.constData(), deckGroup.size() - 1) +
-            QStringLiteral("_Stem0]");
-    stemGroup[stemGroup.size() - 2] = QChar('1' + static_cast<char>(stemIdx));
-    return stemGroup;
+    return QStringView(deckGroup.constData(), deckGroup.size() - 1) +
```

Comment:
last nitpick:
```c++
QString EngineDeck::getGroupForStem(QStringView deckGroup, int stemIdx) {
    DEBUG_ASSERT(stemIdx < 4);
    DEBUG_ASSERT(deckGroup.endsWith(QChar(']')));
    return deckgroup.chopped(1) +
```

Reply:
Unfortunatly this comes witha a runtime overhead. Unlike `QStringView(deckGroup.constData(), deckGroup.size() - 1)`, `deckgroup.chopped(1)` retuns a new allocated QString. It basically boils down to `QString(deckGroup.constData(), deckGroup.size() - 1)`. 

Just learned that that since 6.8 there is a `QString chopped() &&` that will not introduce a new string when used on an rval
`std::move(deckGroup).chopped(1)`
but we have only a `const QString&`

Reply:
> deckgroup.chopped(1) retuns a new allocated QString

Yes, but since my proposal also makes `deckGroup` a `QStringView`, the result is the same. If we know we don't need to store the string we should generally start taking `QStringView`s instead of `const QString&`s IMO. This is a perfect example of a "leaf" function where we know no inner function needs a proper `QString` so this is the place to start such as refactoring.

Reply:
Ah I see. Good idea.